pkg/engine/lifecycletest/resource_reference_test.go

```diff
@@ -1,231 +0,0 @@
-//nolint:goconst
-package lifecycletest
-
-import (
-	"testing"
-
-	"github.com/blang/semver"
-	"github.com/stretchr/testify/assert"
-
-	. "github.com/pulumi/pulumi/pkg/v3/engine"
-	"github.com/pulumi/pulumi/pkg/v3/resource/deploy/deploytest"
-	"github.com/pulumi/pulumi/sdk/v3/go/common/resource"
-	"github.com/pulumi/pulumi/sdk/v3/go/common/resource/plugin"
-)
-
-// TestResourceReferences tests that resource references can be marshaled between the engine, language host,
```

Comment:
The tests in this file are still valid, no? They test the functionality of resource references. The related issue is to remove flags that disable resource references (in case they were broken). I'd enable these tests, as they guard against regressions.

Reply:
Hey @RobbieMcKinstry , I have addressed the requests

Reply:
@RobbieMcKinstry I have addressed the requests